Platformio/HAL/Targets/ESP32/display/display.cpp

```diff
@@ -37,16 +39,40 @@ Display::Display(int backlight_pin, int enable_pin): DisplayAbstract(),
         digitalWrite(this->mEnablePin, LOW);  // LCD Logic on
     }  
 
+    setupTFT();
     setupTouchScreen();
     mFadeTaskMutex = xSemaphoreCreateBinary();
     xSemaphoreGive(mFadeTaskMutex);
 }
 
+void Display::setupBacklight() {
+  // Configure the backlight PWM
+  // Manual setup because ledcSetup() briefly turns on the backlight
+  ledc_channel_config_t ledc_channel_left;
+  ledc_channel_left.gpio_num = (gpio_num_t)mBacklightPin;
+  ledc_channel_left.speed_mode = LEDC_HIGH_SPEED_MODE;
+  ledc_channel_left.channel = LEDC_CHANNEL_5;
+  ledc_channel_left.intr_type = LEDC_INTR_DISABLE;
+  ledc_channel_left.timer_sel = LEDC_TIMER_1;
+  ledc_channel_left.flags.output_invert = 1; // Can't do this with ledcSetup()
+  ledc_channel_left.duty = 0;
+  ledc_channel_left.hpoint = 0;
```

Comment:
Good catch here too I know leaving any part of these structures in IDF causes all sorts of undefined behavior! that is so confusing to track down.